CONTRIBUTING.md

```diff
@@ -1,7 +1,8 @@
 # Contributor documentation
 
-[@infinisil](https://github.com/infinisil) is the admin and main developer of this repository,
-while everybody in [@NixOS/nixpkgs-vet](https://github.com/orgs/NixOS/teams/nixpkgs-vet) has write access.
+[@philiptaron](https://github.com/philiptaron) and
```

Comment:
Q: why the explicit links? I thought GFM created links automatically when \@mentioning a user or team 🙃